modules/vaos/app/services/vaos/systems_service.rb

```diff
@@ -39,6 +39,18 @@ def get_facility_clinics(user, facility_id, type_of_care_id, system_id)
       end
     end
 
+    def get_facility_limits(user, facility_id, type_of_care_id)
+      with_monitoring do
+        url = "/var/VeteranAppointmentRequestService/v4/rest/direct-scheduling/patient/ICN/#{user.icn}/request-limit"
+        url_params = {
+          'institution-code' => facility_id,
+          'clinical-service' => type_of_care_id
+        }
```

Comment:
I think this is fine since there are only two and you're doing the `param` validation in the controller. If there were any more than 2-3, or if we knew we needed to validate on more than just `presence` I might suggest a form object.